app/views/propertyrepresentation/appoint/checkYourAnswers.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.requests.BasicAuthenticatedRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@import config.ApplicationConfig
+@import models.propertyrepresentation.AppointNewAgentSession
+
+@import utils.AppointNewAgentFormatter
+
+@this(govukTable : GovukTable, mainLayout: views.html.mainLayout)
+
+@(data: AppointNewAgentSession)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+@answerMessageKey = @{
+    AppointNewAgentFormatter.format(data.managingProperty.getOrElse(""))
+}
+
+@mainLayout(
+    pageTitle = messages("propertyRepresentation.addAnAgent.title"),
+    backUri = request.headers.get("referer")
+) {
+
+
+
+    <h1 class="govuk-heading-xl">@messages("propertyRepresentation.checkYourAnswers.title")</h1>
+
+    @govukTable(Table(
+        rows = Seq(
+            Seq(
+                TableRow(
+                    content = Text(messages("propertyRepresentation.checkYourAnswers.newAgent"))
+                ),
+                TableRow(
+                    content = Text(data.agentOrganisationName.getOrElse(""))
+                )
+            ),
+            Seq(
+                TableRow(
+                    content = Text(messages("propertyRepresentation.checkYourAnswers.managingProperty"))
+                ),
+                TableRow(
+                    content = Text(messages(answerMessageKey))
+                )
+            )
+        ),
+        head = None,
+        caption = None,
+        firstCellIsHeader = false
+    ))
+
+    <a id="homeLink" class="govuk-button" href="#">@messages("propertyRepresentation.checkYourAnswers.submit")</a>
```

Comment:
just to confirm here, we are using a GET to finalise the submit?

Reply:
we are not submitting anything as we don't have an endpoint from yet